GliaWidgets.podspec

```diff
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   }
   s.exclude_files         = ['GliaWidgets/Window/**']
 
-  s.dependency 'GliaCoreSDK', '1.4.3'
+  s.dependency 'GliaCoreSDK', '1.4.4'
```

Comment:
I suppose similar change should be done in `templates/GliaWidgets.podspec`. Does it?

Reply:
Correct, and it's done